store.go

```diff
@@ -118,3 +207,168 @@ func (b *BucketReaderAt) ReadAt(p []byte, off int64) (n int, err error) {
 
 	return rc.Read(p)
 }
+
+// ErrColNotExist is returned when the column described in the filter does not exist in the columns.
+var ErrColNotExist = errors.New("column does not exist")
+
+func compareMinsMaxes(mins, maxes map[string]*parquet.Value, expr logicalplan.Expr) (bool, error) {
```

Comment:
Same comment as other PR: this seems to be an expression evaluation with a parquet value at first glance (might be missing some complications). Ideally, we would reuse expression code here/write this in a more general way. I might be missing something though